timesketch/lib/sigma_util.py

```diff
@@ -92,11 +97,12 @@ def get_sigma_rules_path():
     return rules_path
 
 
-def get_sigma_rules(rule_folder):
+def get_sigma_rules(rule_folder, sigma_config=None):
     """Returns the Sigma rules for a folder including subfolders.
 
     Args:
         rule_folder: folder to be checked for rules
+        sigma_config: optional: pass a Sigma config object
```

Comment:
fix the docstring, not having `:` twice... 

```
sigma_config: optional dict with Sigma config.
```

or something like that... what is `sigma config`? Is that a special object, is it a dict, is it? unclear from the docstring

Reply:
fixed